CefSharp.OffScreen/ChromiumWebBrowser.cs

```diff
@@ -373,11 +410,29 @@ protected virtual void Dispose(bool disposing)
         }
 
         /// <summary>
+        /// Gets the size of the popup.
+        /// </summary>
+        /// <param name="size">Size of the popup.</param>
+        public void GetPopupSize(ref Size size)
```

Comment:
`out` would be the correct modifier here. I assume your trying to avoid copying the `Struct`? Is this absolutely necessary? Are their other `.Net` frameworks that apply this concept?